trap/src/test/java/com/cursorinsight/trap/transport/TrapHttpTransportTest.kt

```diff
@@ -74,7 +74,7 @@ class TrapHttpTransportTest {
                 readTimeout = 100,
                 compress = false,
             ))
-        transport.send("[[999]]")
+        transport.send("[[999]]", false)
```

Comment:
How about a `send()` method with an optional second argument that defaults to `false`?

Reply:
Done